portfolio/src/main/webapp/script.js

```diff
@@ -64,6 +64,101 @@ function createListElement(text) {
   return liElement;
 }
 
+var mitMarker;
+var dormMarker; 
+
+var mitLatLng;
+var dormLatLng;
+
+var directionsRenderer;
+var directionsService;
+
+// Creating map
+function createMap() {
+    directionsRenderer = new google.maps.DirectionsRenderer();
+    directionsService = new google.maps.DirectionsService();
+
+    mitLatLng = {lat: 42.3601, lng: -71.0942};
+    dormLatLng = {lat: 42.355338, lng: -71.100530};
+
+    const map = new google.maps.Map(document.getElementById('portfolio-map'),
```

Comment:
```
const map = new google.maps.Map(document.getElementById('portfolio-map'), {
  center: {
    lat: (mitLatLng.lat+dormLatLng.lat) / 2, 
    lng: (mitLatLng.lng+dormLatLng.lng) / 2,
  }, 
  mapTypeId: 'terrain',
  zoom: 16,
}); 
```
Always start the object on the prior line, and indent items two spaces over. Also always put a trailing comma on the last item, like above.